pgmax/fg/graph.py

```diff
@@ -1002,7 +1044,14 @@ def get_beliefs(bp_arrays: BPArrays) -> Any:
         )
         return beliefs
 
-    return run_bp, get_bp_state, get_beliefs
+    bp_container = BPContainer(
+        init=functools.partial(update, None),
```

Comment:
Does this mean `init` can only take keyword arguments? If so mention that in the documentation for `init` in `BeliefPropagation`

Reply:
this restriction does not apply, why do you think it would?